### ISSUE TITLE
PuppeteerSharpPdfDemo throws exception 

### ISSUE THREAD

**Issue by imachabeli** [kblok/puppeteer-sharp#294]
using (var browser = await Puppeteer.LaunchAsync(options, Downloader.DefaultRevision))
throws exception 
An attempt was made to transition a task to a final state when it had already completed

**Comment by Meir017** [comment]
can you share a complete code sample?

**Comment by imachabeli** [comment]
```
using System;
using System.Linq;
using System.IO;
using System.Threading.Tasks;
using PuppeteerSharp;

namespace PuppeteerSharpPdfDemo
{
    class MainClass
    {
        static async Task Main(string[] args)
        {
            var options = new LaunchOptions
            {
                Headless = true
            };

            Console.WriteLine("Downloading chromium");
            await Downloader.CreateDefault().DownloadRevisionAsync(Downloader.DefaultRevision);

            Console.WriteLine("Navigating google");
            using (var browser = await Puppeteer.LaunchAsync(options, Downloader.DefaultRevision))
            using (var page = await browser.NewPageAsync())
            {
                await page.GoToAsync("http://www.google.com");

                Console.WriteLine("Generating PDF");
                await page.PdfAsync(Path.Combine(Directory.GetCurrentDirectory(), "google.pdf"));

                Console.WriteLine("Export completed");

                if (!args.Any(arg => arg == "auto-exit"))
                {
                    Console.ReadLine();
                }
            }
        }
    }
}
```

**Comment by kblok** [comment]
That error is being prevented here #255. But it means that we weren't able to execute Chromium.
Could you go to the `.loca-chromium` folder, and try to execute the chrome version which was downloaded there?

**Comment by imachabeli** [comment]
D:\vsprojects\PuppeteerTest\PuppeteerTest\bin\Debug\.local-chromium\Win32-526987\chrome-win32\chrome.exe
Runs fine, chromium opens , complains about missing api keysm not sure if that matters

**Comment by kblok** [comment]
Could you attach a zip with the `.local-chromium` folder?

**Comment by imachabeli** [comment]
https://drive.google.com/open?id=1iWqJXGillG1Z3pd68g8ti_lo_A_6kn9H
Too big, uploaded to google drive

**Comment by kblok** [comment]
@imachabeli downloaded, thanks!

**Comment by kblok** [comment]
Are you using a 32 bit computer @imachabeli?

**Comment by imachabeli** [comment]
nope,  visual studio  by default turns on prefer 32 bit , I unchecked it .  When I run,  64 bit chromium was downloaded successfully , but on the end still got the same error.

**Comment by kblok** [comment]
@imachabeli if you want to use the PDF Export `headless` must be `false`. After setting that property to false, your project worked.

**Comment by kblok** [comment]
Just released V1.0.1. Could you take a look at it?
